AzureCalling/app/src/main/java/com/azure/samples/communication/calling/activities/EndActivity.java

```diff
@@ -37,6 +38,28 @@ protected void onCreate(final Bundle savedInstanceState) {
         });
 
         endTextView = findViewById(R.id.end_return);
-        endTextView.setOnClickListener(l -> finish());
+        endTextView.setOnClickListener(l -> {
+            startIntroActivity();
+            finish();
+        });
+    }
+
+    private void startIntroActivity() {
```

Comment:
We should include private methods after public and protected methods.

Reply:
ok